plugins/module_utils/hmc_resource.py

```diff
@@ -805,3 +805,31 @@ def list_all_lpars_details(self, sys_name, filter=None):
     def getSystemNameFromMTMS(self, system_name):
         attr_dict = self.getManagedSystemDetails(system_name)
         return attr_dict.get('name')
+
+    def copyViosImage(self, sftp_server, sftp_user, sftp_password, image_name):
```

Comment:
this works only for SFTP? this support is not available for FTP or NFS?
If support is available then we have to make the code changes accordingly

Reply:
Added the support for NFS.